client/state/admin-menu/test/menu-fixture.skip.js

```diff
@@ -0,0 +1,483 @@
+export default {
```

Comment:
Is there a reason that we have used a whole set of data? Since we don't validate the data or structure themselves, shall we keep a smaller set of data? so that it is more readable / maintainable?

Reply:
Done and also moved to `fixtures` dir as that avoids it being picked up as a "test" whilst avoiding need to name with `.skip`.